twigextensions/ElixirTwigExtension.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Craft;
+
+use Twig_Extension;
+use Twig_Filter_Method;
+use Twig_Function_Method;
+
+class ElixirTwigExtension extends \Twig_Extension
+{
+    /**
+    * Returns the extension name.
+    * 
+    * @return string
+    */
+    public function getName()
+    {
+        return "Elixir";
+    }
+
+    /**
+    * Returns the extensions filters.
+    * 
+    * @return array
+    */
+    public function getFilters()
+    {
+        return [
+            'elixir' => new \Twig_Filter_Method(
```

Comment:
I might be remembering this wrong, but is the `\` necessary since you're specifically `use`ing the class?